tasks/queue-granules/index.js

```diff
@@ -96,8 +99,8 @@ async function queueGranules(event, testMocks = {}) {
     groupedAndBatchedGranules,
     async (granuleBatchIn) => {
       const collectionConfig = await collectionConfigStore.get(
-        granuleBatchIn[0].dataType,
-        granuleBatchIn[0].version
+        granuleBatchIn[0].dataType !== undefined ? granuleBatchIn[0].dataType : 'placeholder_datatype_value',
+        granuleBatchIn[0].version !== undefined ? granuleBatchIn[0].version : 'placeholder_version_value'
```

Comment:
Does this make collectionConfigStore return undefined?

Reply:
This is used to unqueue the message at the end of the task, I'm not sure downstream tasks will tolerate the collection missing.